### PR TITLE
feat(dataframe): support "__repr__" method in column Series

### PR DESCRIPTION
**Pull request by AChenQ**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-2b4rt2b 实现 "Series.__repr__" 可视化方法](https://app.clickup.com/t/2b4rt2b)

File: graviti/dataframe/column/series.py [hunk 1]
```diff
@@ -85,6 +87,37 @@ def __getitem__(self, key: Union[int, Iterable[int]]) -> Any:
     def __len__(self) -> int:
         return self._data.__len__()
 
+    def __repr__(self) -> str:
+        indices = list(self._get_repr_indices())
+        indice_width = len(max(map(str, indices)))
```
Comment (graczhual) [inline graviti/dataframe/column/series.py:92]: len(str(max(indices)))

File: graviti/dataframe/column/series.py [hunk 2]
```diff
@@ -85,6 +87,37 @@ def __getitem__(self, key: Union[int, Iterable[int]]) -> Any:
...
+
```
Comment (graczhual) [inline graviti/dataframe/column/series.py:93]: do we need to add schema like the example in docstring

File: graviti/dataframe/column/series.py [hunk 3]
```diff
@@ -85,6 +87,37 @@ def __getitem__(self, key: Union[int, Iterable[int]]) -> Any:
...
+            name = item._repr_folding() if hasattr(item, "_repr_folding") else str(item)
+            body.append(name)
+            body_item_width = max(len(name), body_item_width)
+
+        lines = []
+        for indice, value in zip(indices, body):
+            lines.append(f"{indice:<{indice_width+2}}{value:<{body_item_width+2}}")
+        if self.__len__() > _MAX_REPR_ROWS:
+            lines.append(f"...({self.__len__()})")
+        lines.append(f"Series('{self.name}')")
```
Comment (graczhual) [inline graviti/dataframe/column/series.py:107]: if self.name = None, should we print `Series('None')`? pandas series do not print this

Comment (graczhual) [inline graviti/dataframe/column/series.py:107]: Series('category') or name: category like pandas？

File: graviti/dataframe/column/series.py (hunk 2 above)
Comment (AChenQ) [reply graviti/dataframe/column/series.py:93]: We did not integrate schema into the Dataframe and Series now..
Maybe remove schema in example docstring.

**Review by graczhual — APPROVED** [review]

**Review by linjiX — APPROVED** [review]

**Pull request merged**